### ISSUE TITLE
ASM Profile does not get set to blocking when loaded from a file

### ISSUE THREAD

**Issue by f5-applebaum** [F5Networks/f5-appsvcs-extension#544]
* Application Services Version: 3.33.0
 * BIG-IP Version: 15.1

### Summary

When loading an ASM policy with 
                "enforcementMode": "blocking",

from the file method, AS3 does not set policy to "Blocking." 
 

### Steps To Reproduce

1. Submit the following declaration:

Working:
```json
              "CustomWAFPolicy": {
                "class": "WAF_Policy",
                "url": "https://myhost/Rapid_Deployment_Policy_13_1.xml",
                "enforcementMode": "blocking",
                "ignoreChanges": false
              },
```
Not Working:
```json
              "CustomWAFPolicy": {
                "class": "WAF_Policy",
                "file": "/config/cloud/Rapid_Deployment_Policy_13_1.xml",
                "enforcementMode": "blocking",
                "ignoreChanges": false
              },
```

### Expected Behavior
Would like to load the ASM policy and set to blocking from a file instead of from a URL. This is required to avoid ASM making calls out the TMM dataplane (post onboarding when default route is pointing out a TMM interface).

**Comment by dstokesf5** [comment]
AS3 handles the modification of the WAF policy before uploading it to the BIG-IP. When using `file`, the WAF policy is already on the BIG-IP and AS3 does not get an opportunity to modify the data while it is in transit.

Furthermore, we recommend using declarative WAF and updating the enforcementMode there instead of through the AS3 property. Would using declarative WAF work for your use case?

**Comment by amolari** [comment]
@dstokesf5 would the following be possible or file schema is not permitted?

```
              "CustomWAFPolicy": {
                "class": "WAF_Policy",
                "url": "file:///config/cloud/Rapid_Deployment_Policy_13_1.xml",
                "enforcementMode": "blocking",
                "ignoreChanges": false
              },
```

**Comment by dstokesf5** [comment]
It looks like file URLs are not supported. I think that would be an interesting feature request and I am reaching out to the developers to determine the feasibility of that request.

**Comment by dstokesf5** [comment]
In the meantime, could you do something similar by running a temporary http server?
```
cd /config/cloud && python -m SimpleHTTPServer 58080
```
```json
              "CustomWAFPolicy": {
                "class": "WAF_Policy",
                "url": "http://localhost:58080/Rapid_Deployment_Policy_13_1.xml",
                "enforcementMode": "blocking",
                "ignoreChanges": false
              },
```

**Comment by dstokesf5** [comment]
Thank you for your feedback. I have added AUTOTOOL-3015 to our internal product backlog to support file URLs.

**Comment by dstokesf5** [comment]
This issue has been resolved with the 3.37.0 release.
